source/adminguide/reliability.rst

```diff
@@ -61,6 +61,84 @@ still available but the system VMs will not be able to contact the
 management server.
 
 
+Multiple Management Servers Support on agents
+---------------------------------------------
+
+In a Cloudstack environment with multiple management servers, an agent can be
+configured, based on an algorithm, to which management server to connect to.
+This can be useful as an internal loadbalancer or for high availability.
+An administrator is responsible for setting the list of management servers and
+choosing a sorting algorithm using global settings.
+The management server is responsible for propagating the settings to the
+connected agents.
+
+Examples of an agent includes, the process responsible for communication to the
```

Comment:
do you mean by process here?

Reply:
I am trying to describe what I mean by the word "agent". Cloudstack has a few things that are called as such.

Reply:
> I am trying to describe what I mean by the word "agent". Cloudstack has a few things that are called as such.

I think, it is better to be specific about the agents.

Reply:
I've added the process name in brackets, does this make more sense?